packages/fonts/src/test/curatedFonts.js

```diff
@@ -1,44 +0,0 @@
-/*
- * Copyright 2020 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-/**
- * External dependencies
- */
-import { readFileSync } from 'fs';
-import { resolve } from 'path';
-
-/**
- * Internal dependencies
- */
-import { curatedFontNames } from '..';
-
-describe('Curated fonts', () => {
```

Comment:
Maybe I'll revisit this again.

I initially moved the list of curated fonts to the story-editor, but there this test obviously doesn't work.

Perhaps it's better to move it back.